app/src/lib/api.ts

```diff
@@ -226,6 +226,11 @@ export interface IAPIRefStatus {
   readonly statuses: ReadonlyArray<IAPIRefStatusItem>
 }
 
+export interface IAPIBranch {
+  readonly name: string
```

Comment:
Could we document this please? In particular I'd like to see a description of whether this is fully qualified or not.

Reply:
Added in 3581f36e87387ae4fa8beba20ab9a0b0d90c630c